src/parser/serializer.ts

```diff
@@ -1040,7 +918,7 @@ export function serializeInto(
           ignoreUndefined
         );
       } else if (typeof value === 'function' && serializeFunctions) {
-        index = serializeFunction(buffer, key, value, index, checkKeys, depth, serializeFunctions);
+        index = serializeFunction(buffer, key, value, index, checkKeys, depth); // Oh lordy, bug!
```

Comment:
Same mention as above, I can clean up the comment, just noted it as I was fixing the API calls.